auv_control.orogen

```diff
@@ -193,13 +195,15 @@ end
 #
 # It basically projects the (merged) input vector onto the thrusters using the
 # matrix
-task_context "AccelerationController", subclasses: 'Base' do
-
-    # Matrix with size of 6 * n. n means the count of thrusters that are used.
-    # The rows 0 to 2 of the matrix are the linear axis. The lines 3 to 5 of the
+task_context "AccelerationController", subclasses: "Base" do
+    # Matrix with size of n * 6. n means the count of thrusters that are used.
+    # The columns 0 to 2 of the matrix are the linear axis. The columns 3 to 5 of the
     # matrix are the angular axis.
+    #
+    # Warning: By default, the storage orders in Eigen is column-major. So the
+    # thrusterMatrix will be gotten transposed.
```

Comment:
The transposition is a technical detail. I believe it exists so that we do _not_ have to explain what is going on here. A line appears to be a line in the matrix property and all is well. The problem starts when one reads the code (which is where things should be explained)

What you should really explain is what happens if one writes the matrix programatically (as opposed to using the YAML files).

~~~
#
# Warning: if one writes this property programmatically (as opposed to loading it from a YAML
# file), you must transpose it before writing it. See the comments in configureHook for more details
~~~